FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureTableClientFactory.cs

```diff
@@ -50,6 +50,13 @@ public TableClient Create(string tableName)
                 return new TableClient(_tableUrl, tableName);
```

Comment:
as internal connecttion string is supported now, it would be better to merge `StorageEmulatorConnectionString` and `_internalConnectionString`.

Reply:
After offline discussion, we should separate them.